analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -128,6 +138,10 @@ class Maven : PackageManager() {
 
         val vcsFromPackage = maven.parseVcsInfo(mavenProject)
 
+        // If running in SBT mode expect that POM files were generated in a "target" subdirectory and that the correct
+        // project directory is the parent of this.
+        val vcsDir = if (sbtMode) workingDir.parentFile else workingDir
```

Comment:
The `projectDir` argument to this function is now not used anymore. Can we remove it, and rename this local variable to `projectDir` instead?

Reply:
All done.